dagshub/data_engine/model/query.py

```diff
@@ -74,6 +74,13 @@ def column_filter(self) -> Optional[str]:
         return self._column_filter
 
     def compose(self, op: str, other: Optional[Union[str, int, float, "DatasourceQuery", "Datasource"]]):
+        """
+        Compose the current query with another query or a value using the specified operator.
```

Comment:
What does composing mean? Needs a more detailed explanation

Reply:
@kbolashev would this be a good example?